ROMFS/px4fmu_common/init.d/rc.sensors

```diff
@@ -192,7 +192,7 @@ then
 	then
 		# hmc5883 internal SPI bus is rotated 90 deg yaw
 	else
-		if lis3mdl -R 2 start
+		if lis3mdl start
 		then
 			# lis3mdl internal SPI bus is rotated 90 deg yaw
```

Comment:
This comment is invalid now, I guess?

Reply:
Invalid how?

Reply:
"Rotated 90 deg yaw" : is this still correct without the `-R 2` flag?

Reply:
I would agree with you but I did not understand the intentions of the person who initially entered the code. That command did not even work until I added the device the the spi.c. I merely corrected the command so that the compass operated properly on the board. 

Maybe that was a snip-it from somewhere else? I am fine with removing the comment if you concur.